server/command.go

```diff
@@ -698,7 +702,7 @@ func getCommand() *model.Command {
 		DisplayName:      "Jira",
 		Description:      "Integration with Jira.",
 		AutoComplete:     true,
-		AutoCompleteDesc: "Available commands: connect, assign, disconnect, create, transition, view, subscribe, settings, install cloud/server, uninstall cloud/server, help",
+		AutoCompleteDesc: "Available commands: connect, assign, disconnect, create, transition, stats, info, webhook,  view, subscribe, settings, install cloud/server, uninstall cloud/server, help",
```

Comment:
remove stats

Reply:
The following commands are also listed as system administrator commands should they be removed as well including 'stats'?
_Install cloud/server, webhook, subscribe._

Reply:
Yes, thanks for catching these.

Reply:
Thanks I've made the edits. :)

Reply:
@Francois-D also, these descriptions belong in the SysAdmin-specific section, just a few lines below